### ISSUE TITLE
Fieldflow: preselect radio button "checked"

### ISSUE THREAD

**Issue by Borges-Porteous** [wet-boew/GCWeb#1516]
Is there a way to select a radio button on fieldflow initialisation?

The scenario this is being considered in is for a page with a single question to help users complete setting up direct deposit with the CRA, however there are 4 audiences that have different contact info. 95% of visitors are trying to set up direct deposit for personal taxes, so the personal taxes option should be displayed by default, with the radio button checked, but there are 3 others options they can use, if they are a business, trust or non-resident.

**Comment by duboisp** [comment]
That use case is currently not supported by fieldflow, but it is possible to add it by doing some development.

**Comment by Borges-Porteous** [comment]
Ah, that's unfortunate.

Thanks for adding it to the feature requests. Would be great if that could be done in the next version. I assume it wouldn't be a difficult addition to develop?

There are definitely uses cases, where a page is showing information on page load, for example in contextual search results (though admittedly, that's unlikely to use fieldflow) or in these cases where a single question smart page is determining what a Canadian needs to see.

**Comment by duboisp** [comment]
> Would be great if that could be done in the next version.

Yes if someone from the community can submit a PR with that feature including all the related documentation. 

> I assume it wouldn't be a difficult addition to develop?

It depends of the javascript skill of that resource. There is a weekly Tuesday WET-BOEW code sprint where you can get some guidance.
